README.md

```diff
@@ -1 +1,1181 @@
+![Honeycomb](images/result16.png "Honeycomb")
+        
+        
+###### Per resource
+   This will include all self defined span as well.
+   Set the time period to 1 hour as instructed above.
+        
+                  Query parameters use for each category:-
+                 
+                          1. BREAK DOWN - name
+                          2. CALCULATE PER GROUP - COUNT_DISTINCT(traceId), AVG(durationMs)
+                          3. FILTER - name does-not-start-with ballerina/ 
+                          4. LIMIT - 100 
+                          
+                       We filter out the other default ballerina resource using the filter query.
+                               
+![Honeycomb](images/query17.png "Honeycomb")
+                                 
+   The result of the above query is as follows : -
+                     
+![Honeycomb](images/result17.png "Honeycomb")
+
+##### Honeycomb UI Boards
+   
+   These queries can be predefined and added to the board so that the live observability can be achieved without building queries multiple times. 
+   
+   To add a query to a board : -
+      
+   - Create a board.  You can create a board when you run a query. You will see an option “Add to Board” above the query builder.
+   
+![Honeycomb](images/table1.png "Honeycomb") 
+   
+     
+   -  After creating a board select the board. In this guide a board “Requests details” has been already created. Give a name for your query ,add description to be more clear and save the query
+    
+![Honeycomb](images/table2.png "Honeycomb") 
+   
+   - You can view your boards by clicking the “My Boards” in the team’s main menu in honeycomb UI.
+
+![Honeycomb](images/table4.png "Honeycomb")
+
+   
+   -  You can click on any of the boards and run the query for that particular instant.
+   
+![Honeycomb](images/table5.png "Honeycomb")
+
+      
+## About Honeycomb
+The observability is being achieved by sending traces to the honeycomb UI, in which various queries are executed in order to analyse various conditions where the service is being used by the clients. 
+ 
+ Traces refers to the series of the flow of events that occurs when a request is being made and a response is given back. 
+ 
+ 
+![Honeycomb](images/spans2.png "Honeycomb")
+
+
+
+For example a client requesting data from the database as above.
+E refers to an event.
+A trace is the path from E1 to E4.
+
+Traces are further broken down into spans. 
+Spans can be defined as a single operation, i.e server requesting from database to obtain data and receiving it (E2+E3). 
+Spans contain data which can be used for interpreting the performance.
+
+These traces contains metadata (span data) which can be captured by honeycomb and be shown graphically or in raw data.
+
+
+#### Honeycomb open-tracing proxy
+
+Honeycomb works with the data collected in Zipkin format. This proxy will run in your local machine and collects the zipkin formatted trace data and sends to honeycomb. 
```

Comment:
Change to "... This proxy will run in your local machine, collect the Zipkin-formatted trace data, and send it to Honeycomb."